draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2840,14 +2843,15 @@ agent continue to listen for STUN requests and continue to generate
 triggered checks for a media stream, even once processing for that
 stream completes. The rules in this section describe when it is safe
 for an agent to cease sending or receiving checks on a candidate that
-was not selected by ICE, and then free the candidate.
+did not become a selected candidate (is not associated with a selected pair), 
+and then free the candidat.
```

Comment:
"candidate" missing an 'e'

Reply:
Will be fixed.

(BTW, there are a number of spelling errors - I will fix those before submitting the next version)